src/primitives.ts

```diff
@@ -0,0 +1,21 @@
+import fictional from 'fictional'
+
+export const int = fictional.int
```

Comment:
Under the hood we're just aliasing from fictional. This means that we are actually technically supporting fictional's currying and `.options` pattern in the case of the primitive functions - we just don't document it, which I'm hoping is basically the same thing. Happy to just create wrapper functions to avoid people using that API though.